nml/actions/action2var_variables.py

```diff
@@ -231,6 +231,15 @@ def signed_byte_parameter(name, args, pos, info):
     ret = nmlop.AND(args[0], 0xFF, pos).reduce()
     return (ret, [])
 
+def vehicle_railtype(name, args, pos, info):
+    return (expression.functioncall.builtin_resolve_typelabel(name, args, pos, table_name="railtype"), [])
```

Comment:
This commit needs to come after "Codechange: make builtin_resolve_typelabel usable for other action2 variables".